packages/web-client/app/templates/card-pay/balances.hbs

```diff
@@ -1,81 +1,74 @@
               >
-                {{section.cta}}
+                <span>{{section.cta}}</span>
+                {{svg-jar "plus"
+                  class="card-pay-dashboard__balances-section-header-button-icon"
+                  role="presentation"
+                }}
               </Boxel::Button>
-            </:cta>
-            <:disclaimer>
-              {{#if (eq section.workflow 'issue-prepaid-card')}}
-                This is possible if you have a balance of DAI.CPXD in your {{network-display-info "layer2" "fullName"}} wallet.
-                (You can <LinkTo @route="card-pay.token-suppliers">deposit DAI</LinkTo>
-                from your {{network-display-info "layer1" "conversationalName"}} wallet to get a balance of DAI.CPXD in your {{network-display-info "layer2" "fullName"}} wallet.)
-              {{/if}}
-            </:disclaimer>
-          </CardPay::DashboardPanel::Section>
-        {{/each}}
-      </CardPay::DashboardPanel>
-    </section>
-
-    {{#if this.layer2Network.isConnected}}
-      <aside class="card-pay-dashboard__balances" data-test-card-balances>
-        <h3 class="card-pay-dashboard__balances-title">Card Balances</h3>
-
-        {{#if this.prepaidCards}}
-          <section class="card-pay-dashboard__balances-section">
-            <header class="card-pay-dashboard__balances-section-header">
-              <h4 class="card-pay-dashboard__balances-section-title">
-                Prepaid Cards
-                <span class="card-pay-dashboard__balances-count" data-test-prepaid-cards-count>{{this.prepaidCards.length}}</span>
-              </h4>
-
-              {{#let @model.panel.sections.lastObject as |section|}}
-                <Boxel::Button
-                  {{on "click" (set this "flow" section.workflow)}}
-                  @kind="secondary-dark"
-                  @size="small"
-                  @disabled={{section.isCtaDisabled}}
-                  data-test-workflow-button={{section.workflow}}
-                >
-                  <span>{{section.cta}}</span>
-                  {{svg-jar "plus"
-                    class="card-pay-dashboard__balances-section-header-button-icon"
-                    role="presentation"
-                  }}
-                </Boxel::Button>
-              {{/let}}
-            </header>
+            {{/let}}
+          </header>
 
-            <section class="card-pay-dashboard__balances-cards">
-              {{#each this.prepaidCards as |prepaidCard|}}
-                <CardPay::PrepaidCardSafe @safe={{prepaidCard}} />
-              {{/each}}
-            </section>
+          <section class="card-pay-dashboard__balances-cards">
+            {{#each this.prepaidCards as |prepaidCard|}}
+              <CardPay::PrepaidCardSafe @safe={{prepaidCard}} />
+            {{/each}}
           </section>
-        {{/if}}
-      </aside>
+        </section>
+      </section>
     {{/if}}
   </section>
-</section>
```

Comment:
I’m not sure what these changes are about, maybe just rearrangements due to CSS?